.github/workflows/crossplane-upgrade.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: kubectl create namespace crossplane-system
         shell: bash
       - name: Install Crossplane from Stable
-        run: helm repo add crossplane-stable https://charts.crossplane.io/stable && helm repo update && helm install crossplane --namespace crossplane-system crossplane-stable/crossplane --version 1.1.0 --wait
+        run: helm repo add crossplane-stable https://charts.crossplane.io/stable && helm repo update && helm install crossplane --namespace crossplane-system crossplane-stable/crossplane --version 1.9.0 --wait
```

Comment:
any idea why this change is showing up again? I thought we had already merged this into master as part of #24 🤔

Reply:
That's probably because I did not pull the latest commits that were merged to master branch before creating the PR and it show up as another commit when the master branch was merged into logging branch. However, I was not able to squash that commit into the previous commit.